<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoPaciente</class>
 <widget class="QDialog" name="NuevoPaciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo paciente</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="8" column="7">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>RH</string>
       </property>
       <property name="buddy">
        <cstring>cbRH</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="leApellido1"/>
     </item>
     <item row="1" column="3" colspan="3">
      <widget class="QLineEdit" name="leApellido2"/>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>DNI</string>
       </property>
       <property name="buddy">
        <cstring>leDNI</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Telefono</string>
       </property>
       <property name="buddy">
        <cstring>leTlfn</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="leTlfn"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
       <property name="buddy">
        <cstring>leNombre</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDateEdit" name="deNacimiento">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="leProvincia"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Primer apellido</string>
       </property>
       <property name="buddy">
        <cstring>leApellido1</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="7">
      <widget class="QComboBox" name="cbRH">
       <item>
        <property name="text">
         <string>NS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Segundo apellido</string>
       </property>
       <property name="buddy">
        <cstring>leApellido2</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>E-mail</string>
       </property>
       <property name="buddy">
        <cstring>leMail</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="6">
      <widget class="QComboBox" name="cbGrupoSanguineo">
       <item>
        <property name="text">
         <string>NS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fecha de nacimiento</string>
       </property>
       <property name="buddy">
        <cstring>deNacimiento</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Provinvia</string>
       </property>
       <property name="buddy">
        <cstring>leProvincia</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QComboBox" name="cbECivil">
       <item>
        <property name="text">
         <string>Soltero/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Casado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Divorciado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Viudo/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Separado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>En pareja</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Profesión</string>
       </property>
       <property name="buddy">
        <cstring>leProfesion</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Sexo</string>
       </property>
       <property name="buddy">
        <cstring>cbSexo</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="leLocalidad"/>
     </item>
     <item row="8" column="6">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Grupo sangíneo</string>
       </property>
       <property name="buddy">
        <cstring>cbGrupoSanguineo</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>C.P.</string>
       </property>
       <property name="buddy">
        <cstring>leCP</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLineEdit" name="leMail"/>
     </item>
     <item row="10" column="2">
      <widget class="QComboBox" name="cbSexo">
       <item>
        <property name="text">
         <string>Masculino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Femenino</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLineEdit" name="leProfesion"/>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>E. civil</string>
       </property>
       <property name="buddy">
        <cstring>cbECivil</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="leCP"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="leNombre"/>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="leDNI"/>
     </item>
     <item row="10" column="5">
      <widget class="QSpinBox" name="sbHijos"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Localidad</string>
       </property>
       <property name="buddy">
        <cstring>leLocalidad</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Hijos</string>
       </property>
       <property name="buddy">
        <cstring>sbHijos</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Notas</string>
     </property>
     <property name="buddy">
      <cstring>textEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pbCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="iconos.qrc">
       <normaloff>:/new/ico/iconos/close.png</normaloff>:/new/ico/iconos/close.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pbAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
     <property name="icon">
      <iconset resource="iconos.qrc">
       <normaloff>:/new/ico/iconos/checkbox_checked.png</normaloff>:/new/ico/iconos/checkbox_checked.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leNombre</tabstop>
  <tabstop>leApellido1</tabstop>
  <tabstop>leApellido2</tabstop>
  <tabstop>leDNI</tabstop>
  <tabstop>leLocalidad</tabstop>
  <tabstop>leProvincia</tabstop>
  <tabstop>leCP</tabstop>
  <tabstop>leTlfn</tabstop>
  <tabstop>leMail</tabstop>
  <tabstop>leProfesion</tabstop>
  <tabstop>deNacimiento</tabstop>
  <tabstop>cbSexo</tabstop>
  <tabstop>cbECivil</tabstop>
  <tabstop>sbHijos</tabstop>
  <tabstop>cbGrupoSanguineo</tabstop>
  <tabstop>cbRH</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pbAceptar</tabstop>
  <tabstop>pbCancelar</tabstop>
 </tabstops>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCancelar</sender>
   <signal>clicked()</signal>
   <receiver>NuevoPaciente</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
